### DISCUSSION TITLE
BIRT export to Word generates code at the top of a table

### DISCUSSION THREAD

**Discussion by stevehydro** [eclipse-birt/birt#1392] [General]
![table](https://github.com/eclipse-birt/birt/assets/142328454/50e76303-6d93-4e95-b940-88c53aed1109)
Hi,

Using Eclipse IDE Version: Neon.3 Release (4.6.3) Build id: 20170314-1500, when I have a Table Group, in the Table Group Header it adds additional text of {TC "01" \f C \1 "1"}.

It does not appear in any other export format.

How do I remove this?

Thank-you.

**Comment by speckyspooky** [comment]
This is a very strange behavior.
I cannot say how you can remove that because we have over 6 years since this version and try to finalize the version 4.14.0.

If you could try a demo-report I can test it with the latest BIRT version.

**Reply by stevehydro** [reply]
How do I send you a demo report?

**Comment by speckyspooky** [comment]
You can add this here and wrap the report-design-file to a zip-file.
Please don't use complex reports and without external data connections.

**Comment by wimjongman** [comment]
First, upgrade to the latest Eclipse and the latest BIRT. It is impossible to provide support on 7-year-old software.

**Comment by hvbtup** [comment]
This is a Word "field function", or to be precise, a bookmark (IIRC). Usually these fields are not visible. You probably configured Word to show field (with Alt-F9 key?).
The bookmark is the typical result of adding a group to a list or table. When you select a column to define the grouping, this column is automatically added to the "TOC entry" field in the dialog. If you export your report to PDF, for example, you can show the TOC (also known as "outline" in the PDF world)  in the PDF reader application.  If you want to get rid of it, just clear the TOC entry field in the group dialog.
